tests/integration_test_vensim_pathway.py

```diff
@@ -51,15 +51,15 @@ def test_delay_parentheses(self):
         output, canon = runner('test-models/tests/delay_parentheses/test_delay_parentheses.mdl')
         assert_frames_close(output, canon, rtol=rtol)
 
-    def test_delay_pipeline(self):
-        from.test_utils import runner, assert_frames_close
-        output, canon = runner('test-models/tests/delay_pipeline/test_pipeline_delays.mdl')
-        assert_frames_close(output, canon, rtol=rtol)
+    # def test_delay_pipeline(self):
```

Comment:
Is there a reason not to use `@unittest.skip()` here?

Reply:
No, you're right. It's better to use @unittest.skip() here rather than commenting it out. I've added the changes to the pr.